cg/cli/set/cases.py

```diff
@@ -67,18 +65,18 @@ def families(
 
     LOG.info("Would alter cases:")
 
-    for case in cases:
-        LOG.info(case)
+    for case_obj in cases:
+        LOG.info(case_obj)
```

Comment:
I think case is descriptive enough
```suggestion
    for case in cases:
        LOG.info(case_obj)
```